### PR TITLE
Webpack bundle + babel + postcss

### PR DESCRIPTION
**Pull request by evgenTraytyak**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://webpack.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://webpack.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

**Comment by evgenTraytyak** [comment]
I suggest add `CommonsChunkPlugin` to webpack plugins list:
`new webpack.optimize.CommonsChunkPlugin('common_webpack_bundle.js')`

This plugin will be analyse your code and will always create the optimum common file for multiple entry points.

**Comment by evgenTraytyak** [comment]
We can also use this file (`common_webpack_bundle.js`) as entry point.

File: config/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                presets: ['stage-0', 'es2015']
+            },
+            {
+                test: /\.(png|jpg|svg|ttf|eot|woff|woff2)$/,
+                loader: 'file?name=[path][name].[ext]'
+            }
+        ]
+    },
+    postcss: function() {
+        return [require('autoprefixer')];
```
Comment (ezhlobo) [inline config/webpack.config.js]: We should add config for autoprefixer. Suggest to use browserslist in root directory.

Comment (evgenTraytyak) [inline config/webpack.config.js]: :+1:

**Comment by evgenTraytyak** [comment]
- [ ] babel
- [ ] postcss
- [ ] browserlist
- [ ] browserlist
- [ ] react
- [ ] redux
- [ ] babelrc
- [ ] react-transform-hmr
- [ ] react-transform-catch-errors
- [ ] eslint

**Pull request closed without merging**